src/test/java/net/springjfx/test/GuiTest.java

```diff
@@ -0,0 +1,101 @@
+package net.springjfx.test;
```

Comment:
Be aware that this class won't be available in the release jar, so I don't see the point copying it.